### PR TITLE
feat(CSR-1108): update file uploader styles

### PR DESCRIPTION
**Pull request by Ihor-Lavrov**

### CODE REVIEW

File: src/components/file-upload/FileUpload.tsx [hunk 1]
```diff
@@ -7,52 +7,88 @@ import {
...
+  const fileUploaderOverrides = (
+    instructionMessage: string,
+    buttonText: string,
+    preview?: string,
+    error?: boolean,
+  ): BaseFileUploaderOverrides<StyleProps> => ({
+    Root: {
+      style: {
+        height: '100%',
+        background: 'red',
```
Comment (bogdankf1) [inline src/components/file-upload/FileUpload.tsx:40]: @Ihor-Lavrov should it be just `red`, not any value from theme?

File: src/components/file-upload/FileUpload.tsx [hunk 2]
```diff
@@ -7,52 +7,88 @@ import {
...
+        {...props}
+        overrides={{
+          BaseButton: {
+            style: () => ({
+              backgroundColor: 'white',
+              color: error ? negative : dark0,
+              ...margin('14px', '0', '28px'),
+              ...padding('10px', '12px'),
+              ...borderRadius(scale100),
+              fontSize: '14px',
```
Comment (bogdankf1) [inline src/components/file-upload/FileUpload.tsx:64]: @Ihor-Lavrov can we replace these hardcoded px values with theme ones?

File: src/components/file-upload/FileUpload.tsx [hunk 3]
```diff
@@ -7,52 +7,88 @@ import {
...
+      component: preview ? FileUploadPreview : FileUploadInstruction,
+      props: {
+        message: instructionMessage,
+        preview,
+      },
+    },
+    ErrorMessage: {
+      style: {
+        fontSize: '14px',
+        ...margin('0', '0', '12px'),
```
Comment (bogdankf1) [inline src/components/file-upload/FileUpload.tsx:83]: @Ihor-Lavrov and these also

File: src/components/file-upload/FileUpload.tsx (hunk 1 above)
Comment (Ihor-Lavrov) [reply src/components/file-upload/FileUpload.tsx]: fixed

File: src/components/file-upload/FileUpload.tsx (hunk 2 above)
Comment (Ihor-Lavrov) [reply src/components/file-upload/FileUpload.tsx]: fixed

File: src/components/file-upload/FileUpload.tsx (hunk 3 above)
Comment (Ihor-Lavrov) [reply src/components/file-upload/FileUpload.tsx]: fixed

**Review by bogdankf1 — APPROVED** [review]

**Review by hiddeottens — APPROVED** [review]

**Pull request merged**